### PR TITLE
154: Enforce style/practice checks from vt in checkpoint

### LINKED ISSUE #154: Enforce style/practice checks from vt here too

### PR DESCRIPTION
**Pull request by jstrzebonski**
Fixes #154

### CODE REVIEW

File: .github/workflows/check-trailing-whitespace.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  check:
+    name: Find Trailing Whitespace
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: DARMA-tasking/find-trailing-whitespace@master
+        with:
+          exclude: "sketches;lib"
```
Comment (cz4rs) [inline .github/workflows/check-trailing-whitespace.yml]: `sketches` is probably not necessary, I'm not sure about `lib`

**Review by cz4rs — APPROVED** [review]
looks good to me, have you tried to push unsigned / wrongly formatted commits to make sure all works as expected?

File: .github/workflows/check-trailing-whitespace.yml (hunk 1 above)
Comment (jstrzebonski) [reply .github/workflows/check-trailing-whitespace.yml]: Good catch!

Comment (jstrzebonski) [reply .github/workflows/check-trailing-whitespace.yml]: Done.

**Comment by jstrzebonski** [comment]
> 
> 
> looks good to me, have you tried to push unsigned / wrongly formatted commits to make sure all works as expected?

Yeap

**Review by JacobDomagala — APPROVED** [review]

**Review by lifflander — APPROVED** [review]

**Pull request merged**